CouchBaseStorage/Configuration/OrleansDocumentExpiry/OrleansConfigurationSection.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Configuration;
+
+namespace CouchBaseProviders.Configuration.OrleansDocumentExpiry
+{
+    public class OrleansConfigurationSection : ConfigurationSection
```

Comment:
I'm worried using `OrleansConfigurationSection` as it may clash with other implementations (eg official Orleans). How about something like `CouchbaseOrleansConfigurationSection`?